src/front-end/src/components/MentorHome.js

```diff
@@ -1,84 +1,59 @@
+        <h2> Information about &quot;insert selected mentee name&quot;</h2>
+        {/* the form below should change the information depeding on: */}
+        {/* the mentee selected */}
+        <p>
+          Information 1: &quot;insert information1 about selected mentee&quot;
+        </p>
+        <p>
+          Information 2: &quot;insert information2 about selected mentee&quot;
+        </p>
+        <p>
+          Information 3: &quot;insert information3 about selected mentee&quot;
+        </p>
+        {/* button below should link to an appropriate page, once logged out */}
+        <Button onClick={this.Logout} type="primary" htmlType="button">
           Logout
         </Button>
-        <Form>
-          <h1>
-            <center>Welcome back, [Mentor Name from DB]</center>
-          </h1>
-
-          <Form.Item label="Your current mentees" name="userMentees">
-            <Select
-              // so that single options can be selected:
-              mode="single"
-              style={{ width: '100%' }}
-              placeholder="select a mentee"
-              optionLabelProp="label"
-            >
-              {/* the amount of options should be the amount of mentees */}
-              <Option value="Mentee1" label="Mentee1" />
-              <Option value="Mentee2" label="Mentee2" />
-              <Option value="Mentee3" label="Mentee3" />
-            </Select>
-          </Form.Item>
-        </Form>
-        {/* replace what's in square brackets [] w/ info from DB */}
-        <div className="mentee-requests">
-          <h2> Pending Mentee Requests</h2>
-          <p>
-            Click <CheckOutlined className="check" /> to accept a Mentee&apos;s
-            request or <CloseOutlined className="close" /> to decline.
-          </p>
-          <div className="individual-mentee">
-            <table>
-              <tr>
-                <th>
-                  <UserOutlined className="user-icon" />
-                  [Name from DB]
-                  <Button type="text" onClick={this.handleButtonClick}>
-                    <CheckOutlined className="check" id="accept" />{' '}
-                  </Button>
-                  <Button type="text" onClick={this.handleButtonClick}>
-                    <CloseOutlined className="close" id="decline" />{' '}
-                  </Button>
-                </th>
-              </tr>
-              <tr>
-                <td>
-                  <strong> Email: </strong> [Email from Database]
-                </td>
-              </tr>
-              <tr>
-                <td>
-                  <strong> Skills Seeking Mentorship: </strong> [Skills Array
-                  From Database]
-                </td>
-              </tr>
-              <tr>
-                <td>
-                  <strong> Description: </strong> [Description From Database]{' '}
-                </td>
-              </tr>
-            </table>
-          </div>
-        </div>
-      </>
```

Comment:
This is Katrina's code for the mentor UI, we don't want to delete it. To avoid running into this kind of problem, merging develop into your branch frequently may help.

Reply:
done